docs/source/explore/get-started/desktop.rst

```diff
@@ -13,7 +13,7 @@ Desktop users are restricted to using their keyboard and mouse to do things in H
 Movement Controls
 -------------------------------
 
-.. image:: _images/controls-keyboard.png
+.. image:: _images/controls-keyboard.jpg
 
```

Comment:
Under movement, Ctrl + Arrow key also strafes (when enabled). Pretty sure this is new.

Reply:
Done.